### PR TITLE
EyeTracker - add switch case to record a 'message' (tag) in the output

### PR DESCRIPTION
**Pull request by marcobarilari**

### CODE REVIEW

**Comment by marcobarilari** [comment]
will fix #84

**Comment by marcobarilari** [comment]
resolves #84

**Comment by marcobarilari** [comment]
I think those few lines will do what we need to tag the ET output

**Comment by marcobarilari** [comment]
I guess that Travis is complaining because of the new release of `miss_hit`, @remi let me know how you want to manage this at the repo level. 

Easily, I can do it in this pr but we lose a bit of of congruety between pr title and what it really does OR I (or anyonelse) can do it in a separate branch of `dev` and then I rebase this pr with `dev` and we merge this one with only the changes regarding the ET.

LMK

**Comment by Remi-Gau** [comment]
you're right let's focus on the content of this PR.
we'll fix the style in another one.
or maybe just fix the style only of the files you have changed already.

**Comment by Remi-Gau** [comment]
I think miss_hit can do this
```
mh_style filename.m --fix
```

**Comment by marcobarilari** [comment]
done already 🤓

if I remember well, if travis fails merge is blocked, am I right?

**Comment by Remi-Gau** [comment]
nope : you as an admin can override it as long as it is on the `dev` branch

**Comment by Remi-Gau** [comment]
let me rephrase that: NOW that have made you admin and not "just" maintainer you CAN do that. :-)

**Comment by Remi-Gau** [comment]
OK I will review this first. Because the code styling can be automated more easily.

**Review by Remi-Gau — APPROVED** [review]
yup this looks good to me.

that will help us.

**Comment by Remi-Gau** [comment]
merging and then will check the next PR
